stdlib/fcntl.pyi

```diff
@@ -123,6 +124,18 @@ if sys.platform != "win32":
         RWH_WRITE_LIFE_LONG: Final = 4
         RWH_WRITE_LIFE_EXTREME: Final = 5
 
+    if sys.version_info >= (3, 13) and sys.platform == "darwin":
```

Comment:
On my local I can access these on 3.11+, but stubtest fails if we don't have this 3.13+, not sure if anyone has an idea what might be going on. This is at least an improvement in the interim.

Reply:
Whether these constants are defined or not depends on whether they are present on the build machine. In this case it looks like Apple added them at some point, so we should use the same version_info range than for Linux. We need to add stubtest exclude entries, although I would recommend to add it as a regexp entry to the general darwin exclude list (with an appropriate comment, and possibly the current date) as it seems that the availability can fluctuate. I guess that we can remove the allowlist entry in a few years when MacOS systems have been updated.

Reply:
Let me move this to 3.11 and add the ignores.

Reply:
I would agree, but my on my local having the same constant not work on 3.10, but work on 3.11 indicates that something did change between those versions to make them work properly on MacOs. Let me do some more digging.